### PR TITLE
This adds in the gas reporter to our ci

### PR DESCRIPTION
**Pull request by mischat**

### CODE REVIEW

**Review by zajck — DISMISSED** [review]

File: hardhat.config.ts [hunk 1]
```diff
@@ -73,8 +75,12 @@ const config: HardhatUserConfig = {
                 disambiguatePaths: false,
                 runOnCompile: false,
                 strict: true
+        },
+        gasReporter: {
```
Comment (hswopeams) [inline hardhat.config.ts:79]: This and the `contractSizer` config are indented too far

Comment (mischat) [reply hardhat.config.ts:79]: The indentation looks the same as the other config to me, perhaps I am missing something ? @hswopeams ? Can we accept this PR and then i can fix it in a subsequent one?

Comment (mischat) [reply hardhat.config.ts:79]: strange that the linter didn't pick it up... could it be tabs and spaces ....

Comment (mischat) [reply hardhat.config.ts:79]: Screenshot 2021-11-01 at 10 48 52

Comment (mischat) [reply hardhat.config.ts:79]: ok, so it is a tabs vs spaces thing. I have my IDE configured to represent tabs as 4 spaces and to never write out tabs, as tabs can be different on different machines. I will use tabs now, but we should fix this in the future and have our linter take care of this ...

Comment (mischat) [reply hardhat.config.ts:79]: Screenshot 2021-11-01 at 11 04 30

Comment (mischat) [reply hardhat.config.ts]: This should be fixed now @hswopeams

**Review by hswopeams — APPROVED** [review]

**Pull request merged**